TAJO-1245: Add documentation about PostgreSQL and Oracle Catalog driver

jihoonson opened the pull request:
In this patch, I merged the sections for MySQL, MariaDB, PostgreSQL, and Oracle into a single section because most of them are redundant.

hyunsik commented:
There are newly added compilation warnings. Could you fix them?
```
hyunsik@Hyunsiks-MacBook-Pro:tajo-docs$ make html
sphinx-build -b html -d target/doctrees   src/main/sphinx  target/html
Running Sphinx v1.2.1
loading pickled environment... done
building [html]: targets for 49 source files that are out of date
updating environment: 5 added, 34 changed, 6 removed
reading sources... [100%] tsql/variables                                        
/Users/hyunsik/Code/tajo/tajo-8/tajo-docs/src/main/sphinx/configuration/catalog_configuration.rst:59: WARNING: Title overline too short.

=========================
MySQL/MariaDB/PostgreSQL/Oracle Configuration
=========================
/Users/hyunsik/Code/tajo/tajo-8/tajo-docs/src/main/sphinx/configuration/catalog_configuration.rst:59: WARNING: Title overline too short.

=========================
MySQL/MariaDB/PostgreSQL/Oracle Configuration
=========================
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] tsql/variables                                         
/Users/hyunsik/Code/tajo/tajo-8/tajo-docs/src/main/sphinx/jdbc_driver.rst:52: WARNING: unknown document: /configuration/configuration_defaults
writing additional files... genindex search
copying static files... WARNING: html_static_path entry u'/Users/hyunsik/Code/tajo/tajo-8/tajo-docs/src/main/sphinx/static' does not exist
done
copying extra files... done
dumping search index... done
dumping object inventory... done
build succeeded, 4 warnings.

Build finished. The HTML pages are in target/html.
```

jihoonson commented:
Thanks @hyunsik.
I have fixed them all.
Additionally, I fixed a missing backslash character at http://tajo.apache.org/docs/current/tsql/meta_command.html, and a broken link at http://tajo.apache.org/docs/current/jdbc_driver.html#an-example-jdbc-client.

hyunsik commented:
+1 Ship it. Looks good to me.
